model_analyzer/plots/detailed_plot.py

```diff
@@ -115,12 +115,17 @@ def add_measurement(self, measurement):
             be plotted.
         """
 
+        # FIXME: get_parameter no longer exists - need a new function that takes in the model config name/key + PA parameter and returns the value
```

Comment:
is this fixme still valid? Do we need a story?

Reply:
Missed this one! It's already filed TMA-559. I'll change the comment from FIXME to TODO to indicate this.